embedchain/config/QueryConfig.py

```diff
@@ -35,3 +38,15 @@ def __init__(self, template: Template = None):
             and re.search(context_re, template.template)):
             raise ValueError("`template` should have `query` and `context` keys")
         self.template = template
+        if temperature == None:
```

Comment:
please add empty lines between the different parameters. for readibility. thanks.